src/EA.Weee.Web.Tests.Unit/ViewModels/ListOfNotesViewModelBaseTests.cs

```diff
@@ -97,5 +97,71 @@ public void Map_GivenListOfEvidenceNoteData_ShouldReturnMappedData()
             result.EvidenceNotesDataList.Should().NotBeEmpty();
             result.EvidenceNotesDataList.Should().BeEquivalentTo(returnedNotes);
         }
+
+        [Fact]
+        public void Map_GivenCurrentDate_ComplianceYearsListShouldBeReturned()
+        {
+            //arrange
+            var notes = fixture.CreateMany<EvidenceNoteData>().ToList();
+            var model = fixture.Create<ManageEvidenceNoteViewModel>();
+            var date = new DateTime(2022, 1, 1);
+
+            //act
+            var result = testClass.Map(notes, date, model);
+
+            //assert
+            result.ManageEvidenceNoteViewModel.ComplianceYearList.Count().Should().Be(3);
+            result.ManageEvidenceNoteViewModel.ComplianceYearList.ElementAt(0).Should().Be(2022);
+            result.ManageEvidenceNoteViewModel.ComplianceYearList.ElementAt(1).Should().Be(2021);
+            result.ManageEvidenceNoteViewModel.ComplianceYearList.ElementAt(2).Should().Be(2020);
+        }
+
+        [Fact]
+        public void Map_GivenCurrentDateAndManageEvidenceViewModelIsNull_SelectedComplianceYearShouldBeSet()
+        {
+            //arrange
+            var notes = fixture.CreateMany<EvidenceNoteData>().ToList();
+            var date = new DateTime(2022, 1, 1);
+
+            //act
+            var result = testClass.Map(notes, date, null);
+
+            //assert
+            result.ManageEvidenceNoteViewModel.SelectedComplianceYear.Should().Be(2022);
+        }
```

Comment:
would you mind adding a test when manageEvidenceNoteViewModel = null && manageEvidenceNoteViewModel.SelectedComplianceYear <0 ?

Reply:
Not sure I can do that one? When manageEvidenceNoteViewModel  is null I can't set SelectedComplianceYear, unless I'm misunderstanding the test